utensor_cgen/ir/misc/graph_viz.py

```diff
@@ -0,0 +1,31 @@
+from utensor_cgen.ir import OperationInfo, uTensorGraph
```

Comment:
Though after python 3.3, the `__init__.py` is no longer required.
(https://stackoverflow.com/questions/448271/what-is-init-py-for)
But according to the the official python reference, you are using a namespace package rather than regular package.
https://docs.python.org/3.7/reference/import.html#namespace-packages
https://www.python.org/dev/peps/pep-0420/

In our case, I recommend you to add `__init__.py` to make `misc` a regular python module.

Reply:
BTW, python2 should not work with namespace package.